tsconfig.json

```diff
@@ -14,12 +14,11 @@
     "sourceMap": false,
     "declaration": false,
     "noImplicitThis": true,
-    "strict": true,
     "alwaysStrict": true,
-    "noImplicitReturns": true,
-    "noUnusedLocals": false,
     "resolveJsonModule": true,
-    "skipLibCheck": true,
+    "jsx": "preserve",
+    "exactOptionalPropertyTypes": false,
```

Comment:
I do like this one but its worth its on PR, a lot of code currently conflates this which is pretty typical with this settings when trying it out for the first time on a codebase.